buddyboss/buddyboss-platform
# #2006 add filter to skip strip out script

### All Submissions:


### Changes proposed in this Pull Request:
- A sanitization function was protecting display js generated content but its not safe to remove the function
- So, a filter has been added to allow users to skip the sanitization function at their own risk which allows js generated content 

Fixes #2006

### How to test the changes in this Pull Request:

1. Use the filter "bp_privacy_policy_content" to skip using "wp_kses_post" function
2. Go to login page and click privacy policy link below the login form
3. See popup displays the content

### Proof Screenshots or Video
https://www.loom.com/share/d7b89a891e034308baa7044bcab6d4e3

### Other information:
